codevalidator-api/src/main/java/org/sitenv/vocabularies/loader/code/icd10/Icd10CmLoader.java

```diff
@@ -64,21 +74,34 @@ public void load(List<File> filesToLoad) {
 					String available;
 					while ((available = br.readLine()) != null) {
 						
-						String code = available.substring(6, 13).trim();
-						String displayName = available.substring(77).trim();
-					
-						Icd10CmModel model = dbConnection.newInstance(Icd10CmModel.class);;
-						model.setCode(code);
-						model.setDisplayName(displayName);
+						if (pendingCount++ > 0) {
+							insertQueryBuilder.append(",");
+						}
+						
+						insertQueryBuilder.append("(\"");
+						insertQueryBuilder.append(OIOUtils.encode(available.substring(6, 13).trim()));
```

Comment:
OIOUtils.encode() escapes the property values for use in the SQL insert statement.